lib/assets/javascripts/cartodb/models/tabledata.js

```diff
@@ -733,18 +743,34 @@ cdb.admin.CartoDBTableData = cdb.ui.common.TableData.extend({
     }
 
     // use get to fetch the schema, probably cached
-    this._sqlQuery(_.template('select * from (<%= sql %>) __wrapped limit 0')({ sql: sql }), function(data) {
-      // get schema
-      self.query_schema = self._schemaFromQueryFields(data.fields);
-      if (!self.table.isInSQLView()) {
-        if ('the_geom' in self.query_schema) {
-          delete self.query_schema['the_geom_webmercator'];
+    //TODO This is a hacky way to make sure the table data returns before the get schema call.
+    //we need the table schema_alias to before normal schema to inject the alias into the columns
+    var self = this;
+    setTimeout(function(){
```

Comment:
We commited the changes to the branch that should fix things. We tested and did not see any issues.  We will try and fix the linter stuff tomorrow if that in important.